cli/tsc/dts/lib.deno.ns.d.ts

```diff
@@ -2275,6 +2275,9 @@ declare namespace Deno {
      * file.close();
      * ```
      *
+     * @deprecated Use {@linkcode FsFile.writable} instead.
+     * {@linkcode FsFile.write} will be removed in v2.0.0.
+     *
```

Comment:
Where's the issue about these being deprecated?

Reply:
https://github.com/denoland/deno/pull/21465

Reply:
Removing those types seems ok, but removing these runtime functions is going too far, IMO. These functions are useful and easy to use. Removing them is going to break a lot of code.

Reply:
I'm okay with whatever we go with. I just thought this might've been missed with the previous PR. For context, this PR originated from https://github.com/denoland/deno_std/pull/4021#discussion_r1435831354.